src/components/TokenMigration/helpers.ts

```diff
@@ -0,0 +1,38 @@
+import {
+  decodeTransfer,
+  fromTokenBase,
+  getAssetByUUID,
+  hexToString,
+  hexWeiToString,
+  toWei
+} from '@services';
+import { ITokenMigrationConfig, ITxConfig, ITxObject, StoreAccount, TAddress } from '@types';
+
+export const makeTokenMigrationTxConfig = (rawTransaction: ITxObject, account: StoreAccount) => (
```

Comment:
Are you sure we can assume that this is universal for all token migrations?

Reply:
This will be universal for all token migrations as long as we don't implement the ability to migrate less than all of the user's tokens at once.

Reply:
Tx config is just used to show display values :-D